ports/fbthrift/portfile.cmake

```
@@ -0,0 +1,67 @@
+vcpkg_find_acquire_program(FLEX)
+vcpkg_find_acquire_program(BISON)
+
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO facebook/fbthrift
+  REF 3b56786f0822f78a408addffaf3a54b1b8c86dcd # v2019.11.11.00
+  SHA512 d892c6825345d2dc68abbe7a0eacb5ee0444fdea328d8d6bbcd512752058a2de715c03567120090a355115bb9d5d41f3f9c8dc2f82b8054d0b5a2fd1621bf473
+  HEAD_REF master
+)
```

```suggestion
vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO facebook/fbthrift
    REF 3b56786f0822f78a408addffaf3a54b1b8c86dcd # v2019.11.11.00
    SHA512 d892c6825345d2dc68abbe7a0eacb5ee0444fdea328d8d6bbcd512752058a2de715c03567120090a355115bb9d5d41f3f9c8dc2f82b8054d0b5a2fd1621bf473
    HEAD_REF master
)
```
For `REF`, I have the same doubt with the version in CONTROL file.